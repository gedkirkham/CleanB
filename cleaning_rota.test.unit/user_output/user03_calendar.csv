Date,Kitchen,Bedroom,Lounge,Garage,
25/08/2018,Ged,Razvan,Ged,Razvan,
01/09/2018,Razvan,,,,
08/09/2018,Ged,Ged,,,
15/09/2018,Razvan,,,,
22/09/2018,Tom,Razvan,Razvan,Tom,
29/09/2018,Peter,,,,
06/10/2018,Ged,Tom,,,
13/10/2018,Razvan,,,,
20/10/2018,Tom,Peter,Tom,Peter,
27/10/2018,Ged,,,,
03/11/2018,Razvan,Ged,,,
10/11/2018,Ged,,,,
17/11/2018,Razvan,Razvan,Peter,Ged,
24/11/2018,Tom,,,,
01/12/2018,Peter,Tom,,,
08/12/2018,Ged,,,,
15/12/2018,Razvan,Ged,Ged,Razvan,
22/12/2018,Tom,,,,
25/08/2018,Ged,Razvan,,,
01/09/2018,Razvan,,,,
08/09/2018,Ged,Ged,Razvan,Tom,
15/09/2018,Razvan,,,,
22/09/2018,Tom,Razvan,,,
29/09/2018,Peter,,,,
06/10/2018,Ged,Tom,Tom,Ged,
13/10/2018,Razvan,,,,
20/10/2018,Tom,Peter,,,
27/10/2018,Ged,,,,
03/11/2018,Razvan,Ged,Ged,Razvan,
10/11/2018,Ged,,,,
17/11/2018,Razvan,Razvan,,,
24/11/2018,Tom,,,,
01/12/2018,Peter,Tom,Razvan,Ged,
08/12/2018,Ged,,,,
15/12/2018,Razvan,Ged,,,
22/12/2018,Tom,,,,
25/08/2018,Ged,Razvan,Ged,Razvan,
01/09/2018,Razvan,,,,
08/09/2018,Ged,Ged,,,
15/09/2018,Razvan,,,,
22/09/2018,Tom,Razvan,Razvan,Tom,
29/09/2018,Peter,,,,
06/10/2018,Ged,Tom,,,
13/10/2018,Razvan,,,,
20/10/2018,Tom,Peter,Tom,Peter,
27/10/2018,Ged,,,,
03/11/2018,Razvan,Ged,,,
10/11/2018,Ged,,,,
17/11/2018,Razvan,Razvan,Peter,Ged,
24/11/2018,Tom,,,,
01/12/2018,Peter,Tom,,,
08/12/2018,Ged,,,,
15/12/2018,Razvan,Ged,Ged,Razvan,
22/12/2018,Tom,,,,
