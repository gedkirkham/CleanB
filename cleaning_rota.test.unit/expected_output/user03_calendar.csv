Date,Kitchen,Bedroom,Lounge,Garage
18/08/2018,,,,Ged
25/08/2018,Ged,Razvan,Ali,
01/09/2018,Razvan,,,
08/09/2018,Ali,Ali,,
15/09/2018,Ged,,,Razvan
22/09/2018,Razvan,Ged,Ged,
29/09/2018,Ali,,,
06/10/2018,Ged,Razvan,,
13/10/2018,Razvan,,,Ali
20/10/2018,Ali,Ali,Razvan,
27/10/2018,Ged,,,
03/11/2018,Razvan,Ged,,
10/11/2018,Ali,,,Ged
17/11/2018,Ged,Razvan,Ali,
24/11/2018,Razvan,,,
01/12/2018,Ali,Ali,,
08/12/2018,Ged,,,Razvan
